packages/styles/breadcrumb.css

```diff
@@ -21,13 +21,17 @@
 }
 
 .Breadcrumb__Link {
-  font-weight: 400;
+  font-weight: inherit;
```

Comment:
This can be removed entirely, this should be covered by the font-weight properties below.

```suggestion
```